wsf_scraping/pipelines.py

```diff
@@ -68,13 +77,13 @@ def process_item(self, item, spider):
         has_keywords = len(item['keywords'])
 
         # If we need to keep the pdf, move it, else delete it
-        if keep_pdf and has_keywords:
+        if keep_pdf or has_keywords:
```

Comment:
why did you change to or?

Reply:
In both case we want to keep the pdf, either the keep_pdf setting is at 'True' or a keyword was matched in the article

Reply:
👍